KeywordCloudBlockPlugin.inc.php

```diff
@@ -39,6 +39,12 @@ function getDisplayName() {
 		return __('plugins.block.keywordCloud.displayName');
 	}
 
+	/**
+	 * fallBack for getFileCache
+	 */
+	function _cacheDismiss() {
```

Comment:
```suggestion
	function cacheDismiss() {
```
The use of an underscore prefix in method names has no meaning according to [PSR12 coding style](https://www.php-fig.org/psr/psr-12/)(take a look at section 4.4). 

Is there any particular reason that made you use it?

Reply:
It doesn't mean anythinf, thank you for your review 👍🙂

Reply:
> ```suggestion
> 	function cacheDismiss() {
> ```
> The use of an underscore prefix in method names has no meaning according to [PSR12 coding style](https://www.php-fig.org/psr/psr-12/)(take a look at section 4.4). 
> 
> Is there any particular reason that made you use it?  

It doesn't mean anything, thank you for your review 👍🙂

Reply:
Can you send another commit, removing the underscore? So we can merge your contribution? 🙂

Reply:
I've send another pull request for removing the underscore. #18